### PR TITLE
Fixed the CLI to show frequencies.

### PR DESCRIPTION
**Pull request by hoshir**

### CODE REVIEW

**Review by gavinreaney — APPROVED** [review]

File: go.sum [hunk 1]
```diff
@@ -80,6 +80,8 @@ github.com/infostellarinc/go-stellarstation v0.3.0 h1:p2j69fiqTEbRkEtBYeXYtIPbFa
 github.com/infostellarinc/go-stellarstation v0.3.0/go.mod h1:ZUgU88DoZXjmZ2XrIyYfEWlnCV3sYWMkcsRUMvcwNUg=
```
Comment (danada) [inline go.sum:80]: OOC: Why do older versions stick around here?

File: pkg/satellite/pass/list_passes.go [hunk 2]
```diff
@@ -36,8 +36,12 @@ var listPassesVerboseTemplate = []printer.TemplateItem{
 	{"MAX_ELEVATION_TIME", "maxElevationTime"},
 	{"CHANNEL_SET_ID", "channelSet.id"},
 	{"CHANNEL_SET_NAME", "channelSet.name"},
-	{"DL_FREQ_HZ", "downlinkFrequency"},
-	{"UL_FREQ_HZ", "uplinkFrequency"},
+	{"DL_FREQ_HZ", "channelSet.downlink.frequency"},
+	{"DL_MODULATION", "channelSet.downlink.modulation"},
+	{"DL_BITRATE", "channelSet.downlink.bitrate"},
+	{"UL_FREQ_HZ", "channelSet.uplink.frequency"},
```
Comment (danada) [inline pkg/satellite/pass/list_passes.go:42]: If `channelSet.uplink` doesn't exist, will we get a NPE when trying to access modulation, bitrate, and frequency?

Comment (hoshir) [reply pkg/satellite/pass/list_passes.go:42]: That's very good catch. I am going to fix it in a separate PR. Thanks!

File: go.sum (hunk 1 above)
Comment (hoshir) [reply go.sum:80]: I can't find the exact reason why they keep it, but it a behavior of go mod. Though it won't harm you, you can remove it by `go mod tidy` command.
I find a ticket for golang to automatically run go tidy when changing dependencies.

File: pkg/satellite/pass/list_passes.go (hunk 2 above)
Comment (hoshir) [reply pkg/satellite/pass/list_passes.go:42]: Ref. https://github.com/golang/go/issues/32231

**Pull request merged**